### PR TITLE
feat: matic mana balances

### PR DESCRIPTION
**Pull request by cazala**
Show mana L2 balances

### CODE REVIEW

File: src/components/HomePage/HomePage.container.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  getMana,
+  getManaL2,
+} from 'decentraland-dapps/dist/modules/wallet/selectors'
+import { RootState } from '../../modules/reducer'
+import HomePage from './HomePage'
+import { MapStateProps, MapDispatchProps, MapDispatch } from './HomePage.types'
+
+const mapState = (state: RootState): MapStateProps => ({
+  mana: getMana(state) || 0,
+  manaL2: getManaL2(state) || 0,
```
Comment (nachomazzara) [inline src/components/HomePage/HomePage.container.ts:12]: Should we name it `manaMatic` and `getManaMatic` ?

**Review by nachomazzara — COMMENTED** [review]
LGTM. Just a comment related to naming.

**Review by nicosantangelo — APPROVED** [review]
LGTM, we need to resolve https://github.com/decentraland/account/pull/46/files#r575514050 tho, and keep that decision across dapps, because it's also present here https://github.com/decentraland/decentraland-dapps/blob/master/src/containers/UserMenu/UserMenu.container.ts#L28

Also, I know we had this conversation already but L2 is a very convenient name to encompass all the new concepts we're trying to introduce. For example this related issue https://github.com/decentraland/decentraland-dapps/issues/100 proposes `mapped` as a name, as in `mappedProvider` but that's a new concept and the word `mapped` doesn't mean anything in a vacuum.

I know I'm beating a dead horse here but I think we can do better than Matic (and I have the Polygon rename to back me up). If L2 is misleading, isn't there anything else that represents these chains? PoS or sidechain are the two things that come to mind as in `getManaSidechain` and `getSidechainProvider`

We can leave `Matic` here: https://github.com/decentraland/decentraland-transactions/blob/master/src/types.ts#L72 on the type, because in that case the chain id *does* represent that particular tech

**Comment by nachomazzara** [comment]
> LGTM, we need to resolve https://github.com/decentraland/account/pull/46/files#r575514050 tho, and keep that decision across dapps, because it's also present here https://github.com/decentraland/decentraland-dapps/blob/master/src/containers/UserMenu/UserMenu.container.ts#L28
> 
> Also, I know we had this conversation already but L2 is a very convenient name to encompass all the new concepts we're trying to introduce. For example this related issue [decentraland/decentraland-dapps#100](https://github.com/decentraland/decentraland-dapps/issues/100) proposes `mapped` as a name, as in `mappedProvider` but that's a new concept and the word `mapped` doesn't mean anything in a vacuum.
> 
> I know I'm beating a dead horse here but I think we can do better than Matic (and I have the Polygon rename to back me up). If L2 is misleading, isn't there anything else that represents these chains? PoS or sidechain are the two things that come to mind as in `getManaSidechain` and `getSidechainProvider`
> 
> We can leave `Matic` here: https://github.com/decentraland/decentraland-transactions/blob/master/src/types.ts#L72 on the type, because in that case the chain id _does_ represent that particular tech
> 
> Update: What about `(parent)chain` and `childChain` ?
> a good use case to think about is the Wallet object for example, where we have mana and chainId for both versions
> 
> ```ts
> const wallet = {
>     mana,
>     childMana,
>     chainId,
>     childChainId,
> }
> 
> // and a possible
> const MANA_PARENT_TO_CHILD_CHAIN_ID = {
>   [ChainId.ETHEREUM_MAINNET]: [ChainId.MATIC_MAINNET],
>   [ChainId.ETHEREUM_ROPSTEN]: [ChainId.MATIC_MUMBAI],
>   [ChainId.ETHEREUM_GOERLI]: [ChainId.MATIC_MUMBAI]
> }
> ```
> 
> I'd like to say that adding child to mana/chainId is...weird. Maybe we could use an object?
> 
> ```ts
> mana: { parent, child }
> chain: { parent, child }
> ```

I Agreed with what mapped means.

I would prefer going with `MaticMana` and later rename it to polygon which will be a replacement of a string with another, rather than rethink and rename everything in case we need to support other l2 chains which will be renaming/refactor `getSidechainMana` to `getSidechain1Mana` and `getSidechain2Mana`

**Pull request merged**